package pyt.exception;

public class ForbiddenException extends RuntimeException {

}
